seleniumByTechnocredits/src/seleniumPractise/saket/Assignment3.java

```diff
@@ -0,0 +1,73 @@
+package seleniumPractise.saket;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class Assignment3 {
+
+	public static void main(String[] args) throws InterruptedException {
+
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		Thread.sleep(1000);
+
+		driver.manage().window().maximize();
+		driver.findElement(By.id("basicelements")).click();
+		Thread.sleep(2000);
+		WebElement element = driver.findElement(By.xpath("	//button[contains(@id,'javascriptConfirmBox')]"));
+
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		js.executeScript("arguments[0].scrollIntoView(true)", element);
+
+		element.click();
+
+		Alert alert = driver.switchTo().alert();
+
+		alert.accept();
+		Thread.sleep(3000);
+
+		/*
+		 * JavascriptExecutor js1 = (JavascriptExecutor) driver;
```

Comment:
Remove unwanted comments
Always format the code using ctrl+shift+F
Add driver.close at the end .

Code looks good :)

Reply:
thanks :) done with unwated comments